sidebars.json

```diff
@@ -1040,6 +1040,10 @@
             {
               "id": "paas/alarms",
               "type": "doc"
+            },
+            {
+              "id": "paas/deployment-tools",
```

Comment:
Why under paas? Is this something completely related to our paas infrastructure or cìan this documentation be available on any infrastructure?

Reply:
It's not related to our PaaS Infrastructure. Where do you suggest to move it?

Reply:
up

Reply:
let's keep it here; i'd think about a specific "third party integration" category in the infrastructure section

<img width="587" alt="immagine" https://github.com/mia-platform/documentation/assets/7142570/5f1aad2d-ac55-4237-99e5-f6acc13c69e6">


@mia-platform/sig-documentation  what do you think about this?

Reply:
@fredmaggiowski I think you are right, a "third party integration" section could be useful for other tools too (e.g. kube-green). @mia-platform/sig-documentation Can I add this new section?

Reply:
@mia-platform/sig-documentation up, I'd like to close this PR I worked on 😄

Reply:
@akelity

Reply:
@fredmaggiowski I met @akelity and we added the "Third Party Integration" section.